### PR TITLE
Add resize bilinear

### PR DESCRIPTION
**Pull request by axinging**
Also add lint script.

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 PTAL

File: src/kernels/resize_bilinear.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            ${adjustWidth ? 'outShape.z - 1' : 'outShape.z'});
+
+          vec2 effectiveInputOverOutputRatioRC =
+              effectiveInSize / effectiveOutSize;
+
+          // Fractional source index
+          vec2 sourceFracIndexRC = vec2(rc) * effectiveInputOverOutputRatioRC;
+
+          // Compute the four integer indices.
+          ivec2 sourceFloorRC = ivec2(sourceFracIndexRC);
```
Comment (qjia7) [inline src/kernels/resize_bilinear.ts]: I see extra spaces at the end of line. Please remove them.

File: package.json [hunk 2]
```diff
@@ -24,7 +25,7 @@
     "rollup-plugin-node-resolve": "3.3.0",
     "rollup-plugin-typescript2": "0.13.0",
     "rollup-plugin-uglify": "~3.0.0",
-    "tslint": "~5.11.0",
+    "tslint": "^5.17.0",
```
Comment (qjia7) [inline package.json]: You changed the version. So `yarn.lock` will be updated automatically. You need to add that change too.

File: package.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "tsc",
-    "test": "karma start"
+    "test": "karma start",
+    "lint": "tslint -p . -t verbose"
```
Comment (qjia7) [inline package.json]: Sorry, I am not familiar with the package.json grammar. What's the usage of 'lint'? Do you need to update the README.md file?

File: src/kernels/resize_bilinear.ts (hunk 1 above)
Comment (axinging) [reply src/kernels/resize_bilinear.ts]: It seems the previous yarn lint can not detect trailing spaces, so I add no-trailing-whitespace to fix this. PTAL

Comment (qjia7) [reply src/kernels/resize_bilinear.ts]: Thanks Xing. I think `yarn lint` is very helpful. But please make sure that you have resolved all the `yarn lint` error. I see lots of errors in yarn lint. For example:
```
ERROR: (no-any) D:/workspace/jiajia/tfjs/tfjs-webgl2compute/src/backend_webgl2compute.ts[219, 21]: Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.
ERROR: (no-any) D:/workspace/jiajia/tfjs/tfjs-webgl2compute/src/backend_webgl2compute.ts[220, 17]: Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.
ERROR: (no-any) D:/workspace/jiajia/tfjs/tfjs-webgl2compute/src/backend_webgl2compute.ts[220, 49]: Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.
ERROR: (max-line-length) D:/workspace/jiajia/tfjs/tfjs-webgl2compute/src/kernels/conv2d_mm.ts[83, 1]: Exceeds maximum line length of 80
ERROR: (max-line-length) D:/workspace/jiajia/tfjs/tfjs-webgl2compute/src/kernels/conv2d_naive.ts[60, 1]: Exceeds maximum line length of 80
ERROR: (no-any) D:/workspace/jiajia/tfjs/tfjs-webgl2compute/src/kernels/webgl2compute_program.ts[83, 41]: Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.

error Command failed with exit code 2.

Comment (axinging) [reply src/kernels/resize_bilinear.ts]: I didn't edit these files this time. Let's try to fix it in next CL.

Comment (qjia7) [reply src/kernels/resize_bilinear.ts]: It's a little weird for me that you add 'yarn lint' support with many lint errors. It's ok to  split this patch into two. One is or bilinear support. The other is for yarn lint support and with all lint error resolved. How do you think?

Comment (axinging) [reply src/kernels/resize_bilinear.ts]: done

**Comment by qjia7** [comment]
LGTM. Thanks.

**Pull request merged**